HISTORY.md

```diff
@@ -3,11 +3,11 @@
 ## 1.3.0-beta.2 /2019-11-29
 * Update node dependencies.
 * The `@std/esm` package has been deprecated in favour of `esm`. Also `esm` have limited the `.mjs` files to basic functionality without support for esm options, to align with the current experimental support in Node. [More details here](https://github.com/standard-things/esm/issues/696). According to that all `.mjs` files in the `romcal` project have been renamed back to `.js`.
-* Locales management and pickup in romcal have been refactored, and it now behave like moment.js (used by romcal) or the general approach for i18n in app development: locale containing region like `fr-CA` will gracefully fallback to `fr`, if a localisation key isn't in `fr-CA`, or if `fr-CA.js` doesn't exits in the `src/locales` directory. At the end, it always fallback to `en` which is the default language in romcal. Full explanations have been added in the `README.md`.
-* Locales files have been renamed to kebab-case, with capitalized regions, to follow the IETF Language Codes standards, and be close to the general conventions in software development for i18n.
+* Locales management and pickup in romcal have been refactored, and it now behaves like moment.js (used by Romcal) or the general approach for i18n in app development: locale containing region like `fr-CA` will gracefully fall back to `fr` if a localisation key isn't in `fr-CA` or if `fr-CA.js` doesn't exits in the `src/locales` directory. In the end, it always falls back to `en`, which is the default language in Romcal. Full explanations have been added to the `README.md`.
```

Comment:
I’d suggest to change moment.js → Moment.js.

And if you want to revert (to take @pejulian’s decision in account) the naming of romcal to lowercase, there are many places in file. :smiley: